themes/base/theme.css

```diff
@@ -119,7 +119,7 @@ a.ui-button:active,
 .ui-icon-background,
 .ui-state-active .ui-icon-background {
 	border: #003eff/*{borderColorActive}*/;
-	background-color: #ffffff/*{fcActive}*/;
+	background-color: #ffffff/*{bgColorContent}*/;
```

Comment:
Why are we pairing the content background with the active border? Why is this not using the active background?